<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m2DataStatistics</class>
 <widget class="QWidget" name="m2DataStatistics">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>1221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>m2DataStatistics</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="spectrumImageSelector" native="true"/>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="labelImageSelector" native="true"/>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="peakListSelector" native="true"/>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>m/z</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mean</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Std</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>das</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>das</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>d</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>das</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>dasd</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
